velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -417,6 +417,38 @@ TEST_F(ArithmeticTest, isNanDouble) {
   EXPECT_EQ(false, isNan(std::nullopt));
 }
 
+TEST_F(ArithmeticTest, hexWithBigint) {
+  const auto toHex = [&](std::optional<int64_t> value) {
+    return evaluateOnce<std::string>("hex(c0)", value);
+  };
+  EXPECT_EQ(std::nullopt, toHex(std::nullopt));
+  EXPECT_EQ("11", toHex(17));
+  EXPECT_EQ("0", toHex(0));
+  EXPECT_EQ("FFFFFFFFFFFFFFFF", toHex(-1));
+  EXPECT_EQ("7FFFFFFFFFFFFFFF", toHex(INT64_MAX));
+  EXPECT_EQ("8000000000000000", toHex(INT64_MIN));
+}
+
+TEST_F(ArithmeticTest, hexWithVarbinary) {
+  const auto toHex = [&](std::optional<std::string> value) {
+    return evaluateOnce<std::string>("hex(cast(c0 as varbinary))", value);
+  };
+  EXPECT_EQ(std::nullopt, toHex(std::nullopt));
+  EXPECT_EQ("", toHex(""));
+  EXPECT_EQ("537061726B2053514C", toHex("Spark SQL"));
+  EXPECT_EQ("537061726B652153514C", toHex("Spark\x65\x21SQL"));
+}
+
+TEST_F(ArithmeticTest, hexWithVarchar) {
```

Comment:
This test is essentially the same as hexWithVarbinary. Consider combining these by changing the lambda to evaluate 2 expressions, assert that results are the same, then return the result.

Reply:
done.